graphite-ch-optimizer.go

```diff
@@ -82,8 +82,8 @@ type clickHouse struct {
 }
 
 type daemon struct {
-	OneShot      bool          `mapstructure:"one-shot" toml:"one-shot"`
 	LoopInterval time.Duration `mapstructure:"loop-interval" toml:"loop-interval"`
+	OneShot      bool          `mapstructure:"one-shot" toml:"one-shot"`
```

Comment:
Thank you, pretty good trick. Found a description https://stackoverflow.com/a/38034334/3706827